src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php

```diff
@@ -64,6 +64,6 @@ private function getCountryCodes(array $countries)
             $countryCodes[$country->getCode()] = Intl::getRegionBundle()->getCountryName($country->getCode());
         }
 
-        return $countryCodes;
+        return asort($countryCodes);
```

Comment:
Unfortunately, I receive from doctrine only country codes, but I want to sort by country names. However I'm not sure that it is the best solution.